src/v2/view-builder/views/IdentifierView.js

```diff
@@ -16,6 +17,33 @@ const Body = BaseForm.extend({
     return loc('oform.next', 'login');
   },
 
+  saveForm() {
```

Comment:
We have briefly discussed to add general handle to BaseView (e.g. `preSave`) and FormController could invoke it before invoking `idx.proceeded`. Did you get chance to explore such idea?

Reply:
I did! Played around with a couple implementations, but couldn't settle on something scalable. Will discuss this offline with the team during our next sync.